AcadeMeData/tests/test_msgboard_feature.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from AcadeMeData.models import User
+from AcadeMeData.forms import MessageForm
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.mark.django_db
+def test_msgboard_feature(client, user_example):
+    client.force_login(user_example.user)
+    response = client.get("/msgboard/")
+    assert response.status_code == 200
+    form = response.context["form"]
+    assert isinstance(form, MessageForm)
```

Comment:
I'm not sure what this assertion is testing, can you explain?
anyway, you should check that a message inserted by a user appear in the messageboard.

Reply:
Asserting if the form that is being filled in /msgboard/ page is an instance of MessageForm
which is the form that handles messages

BTW 
I have written comments on my code because past experience taught me that it would make the PR approved faster, I do believe it is well written

Reply:
There's no use-case in which the form should not be a MessageForm, so I'm not sure why this assertion is needed. 
A better test would check the fields of the form we expect to see, or that it is saved correctly.